packages/css-jss/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "css-jss",
```

Comment:
why not `jss-css`?

Reply:
because  like react-jss is extension for react,  css-jss is extension for css interface

Reply:
I am not sure though, if this sounds weird we can do the other way around, because one can think of it as of an extension of jss too

Reply:
actually I start thinking it makes more sense as css-jss

Reply:
wdyt @HenriBeck @oliviertassinari

Reply:
I like `css-jss` more, just as it's an interface for `jss` and we usually name these like `-jss`